packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -142,11 +142,25 @@ class Directionality extends InheritedWidget {
 /// This is more efficient than adding and removing the child widget from the
 /// tree on demand.
 ///
+/// ## Opacity Animation
+///
+/// `Opacity` animations should be built using [AnimatedOpacity] rather than
```

Comment:
[Opacity]

Reply:
@Hixie to make sure I understand the practice here, the square bracket linked syntax is to be used for each and every mention of a linkable item?  Historically when I've written docs I link the first mention in a section and then simply apply syntax highlighting, so I want to make sure I understand the practices correctly.

Reply:
Yup, we want as many links as possible. That way people can immediately click the link and don't have to hunt around for the one link for that word. It also makes it a lot easier for us to copy-edit later, we don't have to keep checking that we didn't remove the one link or add a redundant link.

See also https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-comments-dartdocs for other advice (not that it covers this particular detail).

Reply:
I'd refer to both [FadeTransition] and [AnimatedOpacity]